src/bundle/Resources/public/js/scripts/helpers/backdrop.helper.js

```diff
@@ -0,0 +1,34 @@
+(function (global, doc, ibexa) {
+    class Backdrop {
+        constructor(config = {}) {
+            this.isTransparent = config.isTransparent ?? false;
+            this.extraClasses = config.extraClasses ?? [];
+            this.backdrop = null;
+
+            this.remove = this.remove.bind(this);
+            this.init = this.init.bind(this);
+        }
+
+        remove() {
+            if (this.backdrop) {
+                this.backdrop.remove();
+            }
+        }
+
+        init() {
+            const classes = {
+                'ibexa-backdrop--transparent': this.isTransparent,
+            };
+            const backdropClasses = Object.keys(classes).filter((property) => classes[property]);
```

Comment:
it seems like you want to have some `createCssClassNames` we have in react, maybe it is time to make this helper global?

Reply:
It it global helper but I do not want make single string just array of them.
I could use createCssClassNames and than split them but what the point.